src/Controller/OrderController.php

```diff
@@ -254,9 +255,16 @@ public function indexAction(Request $request,
             }
         }
 
+        //FEAT:
+        $nonprofits =  $this
+            ->getDoctrine()
+            ->getRepository(Nonprofit::class)
+            ->findBy(['enabled'=>true], ['id' => 'ASC']); ;
+
         return $this->render('order/index.html.twig', array(
             'order' => $order,
             'form' => $form->createView(),
+            'nonprofits' => $nonprofits,
```

Comment:
This is not necessary to load the entities here, as they are already loaded by the `EntityType` in `CheckoutAddressType`. You should use a [form theme](https://symfony.com/doc/5.3/form/form_themes.html) custom block, and you will find the entities there, in the `form` variable.